routes/soldiers.js

```diff
@@ -0,0 +1,75 @@
+import events from 'events';
+import { connectToDB, client } from '../connections.js';
+import { addNewSoldier, lookForSoldier, lookForAllSoldiers } from '../database.js';
+
+const eventEmitter = new events.EventEmitter();
+eventEmitter.on('load', connectToDB);
+
+const logOptions = {
```

Comment:
this is not the soldier business!

Reply:
accidentally been in both soldier and app.
fixed